amenity-points.mss

```diff
@@ -2064,11 +2064,12 @@
     text-wrap-width: @standard-wrap-width;
   }
 
-  [feature = 'natural_tree'][zoom >= 17] {
+    [feature = 'natural_tree'] {
     text-name: "[name]";
     text-size: @standard-text-size;
     text-fill: green;
-    text-dy: 7;
+    [zoom >= 16][denotation = 'natural_monument'] { text-dy: 6; }
```

Comment:
I was thinking it might be in there as looking at other stuff I see some stuff get changed into features, but i am not to sure how to go by doing it just yet so help is appreciated.